File: src/main/java/com/sothawo/springdataelastictest/presidents/Term.java
/*
 * (c) Copyright 2021 sothawo
 */
package com.sothawo.springdataelastictest.presidents;

import org.springframework.data.elasticsearch.annotations.Field;

/**
 * @author P.J. Meisch (pj.meisch@sothawo.com)
 */
record Term(
    @Field(name = "gte")
    Integer from,
    @Field(name = "lte")
    Integer to
) {
}

File: src/main/kotlin/com/sothawo/springdataelastictest/presidents/Term.kt
/*
 * (c) Copyright 2021 sothawo
 */
package com.sothawo.springdataelastictest.presidents;

import org.springframework.data.elasticsearch.annotations.Field;

/**
 * @author P.J. Meisch (pj.meisch@sothawo.com)
 */
data class Term(
    @Field(name = "gte")
    val from: Int,
    @Field(name = "lte")
    val to: Int
)
